types/react-relay/v1/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for react-relay 1.3
```

Comment:
If you're adding a newer version, that should go directly in `types/react-relay` and the old definitions should go in `types/react-relay/v0`, or be deleted if you probably won't be updating them.

Reply:
Got rid of those.